### ISSUE TITLE
Lookahead Bias

### ISSUE THREAD

**Issue by ukuarep** [jinglescode/time-series-forecasting-pytorch#19]
The Normalizer class computes the mean and standard deviation over the full data set.

Doesn't this introduce lookahead bias in the model essentially feeding information about future stock prices into you past price data?

**Comment by ghost** [comment]
This is correct ^^. Also, in the preprocessing steps, shuffling the data -> dataloader arg leads to lookahead bias...
